src/java.base/share/classes/jdk/internal/org/objectweb/asm/Frame.java

```diff
@@ -611,11 +611,11 @@ private int pop() {
       */
     private void pop(final int elements) {
         if (outputStackTop >= elements) {
-            outputStackTop -= elements;
+            outputStackTop -= (short)elements;
         } else {
```

Comment:
I think we usually try to avoid changing the ASM code if possible but maybe you have to do this because the compiler option is used for compiling java.base?